iree/turbine/kernel/compiler/kernel_codegen.py

```diff
@@ -120,7 +125,7 @@ def sym_to_dim_asm(s: IndexSymbol) -> str:
                 # Unranked. Not well supported, but for completeness.
                 spec_asm = element_type_asm
             strides = strides_from_symbolic_shape(idx_context, kb_t.symbolic_shape)
-            if strides is None:
+            if strides is None or _is_symbolic(strides):
```

Comment:
If we have a symbolic (i.e. dynamic) strides, shouldn't memref type be `memref<{spec_asm}, strided<[?, ?, ...], offset: ?>`?

Reply:
Good point, I will fix this.

Reply:
done